### PR TITLE
Implementation of methods that allow the user to create one or more s…

### PR DESCRIPTION
**Pull request by Fatimata-dev**
Implementation of methods that allow the user to create one or more shipments and add one or more tasks to their shipments.
renaming of certain attributes to be consistent with the back.
Creation of a pipe to filter the tasks instead of the previous filter

### CODE REVIEW

File: src/app/components/task/task.component.ts [hunk 1]
```diff
@@ -14,17 +13,15 @@ import { TaskService } from '../../service/task.service';
...
 
   @Input() public task !: Task;
   
   @Output()
   deleteTaskToParent: EventEmitter<Task> = new EventEmitter();
 
   toggleTaskComplete(){
-    this.task.completed = !this.task.completed;
-    this.taskService.updateTask(this.task);
+    //this.taskService.toggleTaskComplete(this.task);
```
Comment (LucieLDa) [inline src/app/components/task/task.component.ts:24]: Dead code

File: src/app/components/task/task.component.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <div class="task-container">
     <label class="checkbox-container">
-        <input type="checkbox" [checked]="task.completed" (change)="toggleTaskComplete()"/> 
+        <!-- <input type="checkbox" [checked]="task.isCompleted()" (change)="toggleTaskComplete()"/>  -->
```
Comment (LucieLDa) [inline src/app/components/task/task.component.html:3]: Dead code, and doesn't putting this line in comment remove the ability to check a task ?

File: src/app/service/tasks/task.service.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+     return this.httpClient.delete<Task>(`${this.apiUrl}/expeditions/${expId}/tasks/${taskId}`)
+  }
+
+  // toggleTaskComplete(task : Task) : void {
+  //   this.tasks.filter( (t, index) => {
+  //     if(task.id === t.id) t.changeCompleted();
+  //   });
+  //   this.taskUpdated.emit(this.tasks);
+  // }
+}
```
Comment (LucieLDa) [inline src/app/service/tasks/task.service.ts:30]: dead code

**Review by LucieLDa — CHANGES_REQUESTED** [review]

File: src/app/components/task/task.component.scss [hunk 4]
```diff
@@ -168,7 +168,7 @@
     bottom: 4px;
 }
 
-.priorite{
+.priority{
```
Comment (LucieLDa) [inline src/app/components/task/task.component.scss:171]: You forgot to rename the class in html file from priorite to priority

**Review by LucieLDa — CHANGES_REQUESTED** [review]

**Review by Benja-san — APPROVED** [review]

**Review by LucieLDa — APPROVED** [review]

**Pull request merged**